app/controllers/message_threads/bulk/message_drafts_controller.rb

```diff
@@ -8,7 +8,8 @@ def submit
 
         message_threads = message_thread_policy_scope.where(id: ids).includes(:messages)
         message_threads.transaction do
-          if Govbox::SubmitMessageDraftsAction.run(message_threads)
+          submission_results = [Govbox::SubmitMessageDraftsAction.run(message_threads), Fs::SubmitMessageDraftsAction.run(message_threads)].flatten
```

Comment:
Tato logika by imho mala byt v tom govbox action, ten nech rozhoduje ako sa to ma odoslat. (Ci cez upvs/fs/...)